bundles/ch.elexis.laborimport.synlab/src/ch/elexis/laborimport/synlab/PreferencePage.java

```diff
@@ -21,9 +21,9 @@
 
 public class PreferencePage extends FieldEditorPreferencePage implements IWorkbenchPreferencePage {
 
-	public static final String JAR_PATH = "synlab/jar_path"; //$NON-NLS-1$
-	public static final String INI_PATH = "synlab/ini_path"; //$NON-NLS-1$
-	public static final String DL_DIR = "synlab/downloaddir"; //$NON-NLS-1$
+	public static final String JAR_PATH = "medisyn/jar_path"; //$NON-NLS-1$
+	public static final String INI_PATH = "medisyn/ini_path"; //$NON-NLS-1$
+	public static final String DL_DIR = "medisyn/downloaddir"; //$NON-NLS-1$
```

Comment:
Hier werden die parameter Namen in der Konfig Tabelle geändert, dadurch geht die bestehende Konfiguration verloren.